src/model/bpmn/shape/ShapeUtil.ts

```diff
@@ -31,4 +36,7 @@ export default class ShapeUtil {
   public static topLevelBpmnEventKinds(): ShapeBpmnElementKind[] {
     return this.EVENTS_KIND;
   }
+  public static topLevelBpmnGatewayKinds(): ShapeBpmnElementKind[] {
```

Comment:
:question: could we rename it into `gatewayKinds` for simplicity? in #208, we introduce `taskKinds`.
For the `events`, the `top level` reference is to try to distinguish the list from the one `ShapeBpmnEventKind#supportedBpmnEventKinds`

Reply:
yes, good point